include/networkit/matching/BMatcher.hpp

```diff
@@ -0,0 +1,39 @@
+#ifndef NETWORKIT_MATCHING_B_MATCHER_HPP_
+#define NETWORKIT_MATCHING_B_MATCHER_HPP_
+
+#include <networkit/base/Algorithm.hpp>
+#include <networkit/matching/BMatching.hpp>
+
+namespace NetworKit {
+
+/**
+ * @ingroup matching
+ * Abstract base class for b-matching algorithms.
```

Comment:
This class isn't abstract anymore, you could remove the override of `run()` to accomplish this.

Reply:
Thanks for pointing to this. Will fix it.